lib/googleHelpers.js

```diff
@@ -0,0 +1,60 @@
+const { OAuth2Client } = require('google-auth-library')
+const helpers = require('./helpers')
+
+// constants
+const PA_CLIENT_ID = helpers.getEnvVar('PA_CLIENT_ID')
+const PA_CLIENT_SECRET = helpers.getEnvVar('PA_CLIENT_SECRET')
```

Comment:
Are these new environment variables? If so, could you please update the `.env.example` files in `brave-alert-lib`, `BraveSensor` and `BraveButtons`?

Reply:
Done!